package.json

```diff
@@ -14,6 +14,160 @@
+    "test:walkontable": "npm run build:walkontable && npm run test:walkontable.dump && npm run test:walkontable.puppeteer",
+    "test:walkontable.random": "npm run build:walkontable && npm run test:walkontable.dump && npm run test:walkontable.puppeteer -- --random",
+    "test:walkontable.watch": "node ./.config/bin/trigger-on-stdout-change.js \"concurrently --raw --kill-others \\\"npm run build:walkontable -- --watch\\\" \\\"npm run test:walkontable.dump -- --watch\\\"\" \"npm run test:walkontable.puppeteer\"",
+    "test:walkontable.dump": "cross-env-shell BABEL_ENV=commonjs_e2e NODE_ENV=test-walkontable env-cmd -f ./hot.config.js webpack --hide-modules helpers=./src/3rdparty/walkontable/test/helpers/index.js specs=./src/3rdparty/walkontable/test/spec/index.js",
+    "test:walkontable.puppeteer": "node test/scripts/run-puppeteer.js src/3rdparty/walkontable/test/SpecRunner.html",
+    "test:production": "npm run build:umd.min && npm run build:languages.min && npm run test:production.dump && npm run test:e2e.puppeteer",
+    "test:production.random": "npm run build:umd.min && npm run build:languages.min && npm run test:production.dump && npm run test:e2e.puppeteer -- --random",
+    "test:production.dump": "cross-env-shell BABEL_ENV=commonjs_e2e NODE_ENV=test-production env-cmd -f ./hot.config.js webpack --hide-modules helpers=./test/helpers/index.js e2e=./test/e2e/index.js",
+    "test:e2e": "npm run build:umd && npm run build:languages && npm run test:e2e.dump && npm run test:e2e.puppeteer",
+    "test:e2e.random": "npm run build:umd && npm run build:languages && npm run test:e2e.dump && npm run test:e2e.puppeteer -- --random",
+    "test:e2e.verbose": "npm run build:umd && npm run build:languages && npm run test:e2e.dump && npm run test:e2e.puppeteer -- --verbose",
+    "test:e2e.watch": "node ./.config/bin/trigger-on-stdout-change.js \"concurrently --raw --kill-others \\\"npm run build:umd -- --watch\\\" \\\"npm run test:e2e.dump -- --watch\\\"\" \"npm run test:e2e.puppeteer\"",
+    "test:e2e.dump": "cross-env-shell BABEL_ENV=commonjs_e2e NODE_ENV=test-e2e env-cmd -f ./hot.config.js webpack --hide-modules helpers=./test/helpers/index.js e2e=./test/e2e/index.js",
+    "test:e2e.puppeteer": "node test/scripts/run-puppeteer.js test/E2ERunner.html",
+    "test:unit": "cross-env-shell npm_config_testPathPattern=. BABEL_ENV=commonjs env-cmd -f ./hot.config.js jest --testPathPattern=$npm_config_testPathPattern",
+    "test:unit.watch": "cross-env-shell BABEL_ENV=commonjs env-cmd -f ./hot.config.js jest --testPathPattern=$npm_config_testPathPattern --watch",
+    "test:mobile.dump": "cross-env-shell BABEL_ENV=commonjs_e2e NODE_ENV=test-mobile env-cmd --no-override -f ./hot.config.js webpack --hide-modules helpers=./test/helpers/index.js mobile=./test/e2e/mobile/index.js",
+    "test:types": "tsc -p ./test/types",
+    "test:ci": "node scripts/run-targeted-tests.js",
```

Comment:
The script `npm run test:ci` is supposed to be run on CI? If true, have you considered that right now the tests are manually separated into two pipelines `npm run test:production` and `npm run test:unit && npm run test:types && npm run test:walkontable && npm run test:e2e`? 

Are we able to do that the same with the new script (parallel the tasks)?

Reply:
I forgot about that 😨  I'll try to figure something out, but it might be kind of tricky, as the wrappers have just one test "pipeline" each.